pandas/tests/frame/test_apply.py

```diff
@@ -1367,3 +1367,96 @@ def test_consistency_of_aggregates_of_columns_with_missing_values(self, df, meth
         tm.assert_series_equal(
             none_in_first_column_result, none_in_second_column_result
         )
+
+
+class TestDataFrameNamedAggregate:
+
+    # GH 26513
+    def test_agg_relabel(self):
```

Comment:
it looks like this is testing several cases of increasing complexity, is that correct?  if so, can you split them into separate tests with informative names

Reply:
separated!